microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -160,35 +155,47 @@ In the following tables, features are marked with the following statuses:
 [id="microshift-4-15-bug-fixes"]
 == Bug fixes
 
-//[discrete]
+[discrete]
 [id="microshift-4-15-installation-bug-fixes"]
 === Installation
 * Previously, the `microshift-release-info` RPM depended on the `microshift` RPM. With this release, you can download and install the `microshift-release-info` RPM independently for use in image building. (link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
 
-//[discrete]
-//[id="microshift-4-15-storage-bug-fixes"]
-//==== Storage
+* Previously, the Greenboot health check script printed outputs for some checks that were not picked up by `journald`, resulting in missing log entries when running the `journalctl -u greenboot-healthcheck` command. With this release, the production of logs by the Greenboot healthcheck has been fixed so that all outputs are linked to the `systemd` unit, making them easily available to read. (link:https://issues.redhat.com/browse/OCPBUGS-20037[*OCPBUGS-20037*])
 
+* Previously, the {microshift-short} clean-up script failed if CRI-O services were not present or not running. This prevented {microshift-short} and its dependencies from fully uninstalling. It also prevented the running of the clean-up script on a system with a new install of the `microshift-rpm` that had not started. With this release, the {microshift-short} clean-up script skips the CRI-O steps if the service is not running, reports that status, and continues with clean-up activities. (link:https://issues.redhat.com/browse/OCPBUGS-22936[*OCPBUGS-22936*])
 
-//[discrete]
-//[id="microshift-4-15-running-applications-bug-fixes"]
-//==== Running applications
-
+* Previously, the Greenboot health check reported a `RED` status when the Logical Volume Manager Storage (LVMS) plug-in was disabled because no volume groups (VGs) were present. Because volume groups are not required for {microshift-short}, the health check should report `GREEN` without VGs. Now, when LVMS is disabled, the Greenboot health check skips the check for the `openshift-storage` namespace and reports `GREEN` status. (link:https://issues.redhat.com/browse/OCPBUGS-25689[*OCPBUGS-25689*])
```

Comment:
```suggestion
* Previously, the Greenboot health check reported a `RED` status when the Logical Volume Manager Storage (LVMS) component was disabled because no volume groups (VGs) were present. Because volume groups are not required for {microshift-short}, the health check should report `GREEN` without VGs. Now, when LVMS is disabled, the Greenboot health check skips the check for the `openshift-storage` namespace and reports `GREEN` status. (link:https://issues.redhat.com/browse/OCPBUGS-25689[*OCPBUGS-25689*])
```